api/test/integration/projects/projects-key-costs.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpStatus } from '@nestjs/common';
+import { projectsContract } from '@shared/contracts/projects.contract';
+import { TestManager } from 'api/test/utils/test-manager';
+
+describe('Projects key costs', () => {
+  let testManager: TestManager;
+  let jwtToken: string;
+
+  beforeAll(async () => {
+    testManager = await TestManager.createTestManager();
+    const result = await testManager.setUpTestUser();
+    jwtToken = result.jwtToken;
+
+    await testManager.ingestCountries();
+  });
+
+  afterEach(async () => {
+    await testManager.clearDatabase();
+  });
+
+  afterAll(async () => {
+    await testManager.close();
+  });
+
+  describe('ProjectsController', () => {
+    it('should return 200 and the key costs for the filtered projects dataset', async () => {
+      // Given
+      await testManager.ingestProjectScoreCards(jwtToken);
+      await Promise.all([
+        testManager.mocks().createProject(),
+        testManager.mocks().createProject(),
+        testManager.mocks().createProject(),
+      ]);
+
+      // When
+      const projectsResponse = await testManager
+        .request()
+        .get(projectsContract.getProjects.path)
+        .query({ 'sort[0]': ['projectName'] });
+      const keyCostsResponse = await testManager
+        .request()
+        .get(projectsContract.getProjectsKeyCosts.path)
+        .query({ 'sort[0]': ['projectName'] });
+
+      // Then
+      expect(projectsResponse.status).toBe(HttpStatus.OK);
+      expect(keyCostsResponse.status).toBe(HttpStatus.OK);
+
+      const projects = projectsResponse.body.data;
+      const projectsKeyCosts = keyCostsResponse.body.data;
+
+      expect(projects.length).toBe(projectsKeyCosts.length);
+      expect(projects[0].projectName).toBe(projectsKeyCosts[0].projectName);
+      expect(projects[1].projectName).toBe(projectsKeyCosts[1].projectName);
+      expect(projects[2].projectName).toBe(projectsKeyCosts[2].projectName);
+    });
```

Comment:
This test is correct, but I think we are missing the most valuable test for this specific implementation, which is to assert that the response contains the **required set of costs**, since these are not dynamic and they have to be always the same.

Please add that one so that if, for any reason, we fuck up re-modeling the data/column names, etc... We realise that the response no longer includes what it should